common-ui/src/main/res/color/button_primary_container_selector.xml

```diff
@@ -17,6 +16,6 @@
   -->
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="@color/subtleGrayTwo" android:state_enabled="false" />
-    <item android:color="@color/cornflowerBlue" />
-</selector>
+    <item android:color="?attr/daxColorButtonPrimaryContainer" android:state_enabled="true"/>
+    <item android:color="?attr/daxColorContainerDisabled"/>
```

Comment:
Same question as before, don't we need `android:state_enabled="false"`?

Reply:
Ditto. No, this is a backgroundTint. I've just slightly modified the default [MaterialButton backgroundTint](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/button/res/color/mtrl_btn_bg_color_selector.xml)